webpack.config.dev.js

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
Is there any way we can bundle this in wp-scripts and offer support for third-party plugin devs too?

Reply:
Definitely, I just wanted to stabilize it first.

Should I bundle it in in this PR or wait for more people who develop Gutenberg to use it and report potential issues?

Reply:
Ok, sounds like a reasonable plan to me.  

cc @gziolo for awareness too.

Reply:
Nice one. It’s super exciting to see that it can be integrated into Gutenberg 👍